src/openassetio-python/cmodule/src/managerApi/ManagerInterfaceBinding.cpp

```diff
@@ -247,6 +257,10 @@ void registerManagerInterface(const py::module& mod) {
            py::arg("context").none(false), py::arg("hostSession").none(false),
            py::arg("successCallback"), py::arg("errorCallback"),
            py::call_guard<py::gil_scoped_release>{})
+      .def("entityTraits", &ManagerInterface::entityTraits, py::arg("entityReferences"),
+           py::arg("access"), py::arg("context").none(false), py::arg("hostSession").none(false),
```

Comment:
[Question] I see that it's consistent, but why are the accesses not named their specific names rather than just `access` here?

Reply:
Good spot - seems like this slipped through the net with resolve. We generally are supposed to match the c++ sig arg names I think.

Reply:
Yeah good spot, will ninja a prior commit to address this for the other methods, and then fixup `entityTraits`